pkg/controller/deployment/util/deployment_util.go

```diff
@@ -949,3 +949,18 @@ func GetDeploymentsForReplicaSet(deploymentLister appslisters.DeploymentLister,
 
 	return deployments, nil
 }
+
+// ReplicaSetsByRevision sorts a list of ReplicaSet by revision, using their creation timestamp or name as a tie breaker.
```

Comment:
nit: I don't see name being used as a tie breaker

Reply:
> nit: I don't see name being used as a tie breaker

while, actually the func "controller.ReplicaSetsByCreationTimestamp" itself implements the log of using name as a tie breaker 
https://github.com/kubernetes/kubernetes/blob/c8f98589208e8c5a0d9262ebd90985acc55d2d5a/pkg/controller/controller_utils.go#L970-L980